doc/source/serve/advanced.rst

```diff
@@ -254,6 +254,29 @@ errored requests through the Ray metrics monitoring infrastructure. By default,
 the metrics are exposed in Prometheus format on each node. See the
 `Ray Monitoring documentation <../ray-metrics.html>`__ for more information.
 
+Dependency Management
+=====================
+
+Ray Serve supports serving backends with different (possibly conflicting)
+dependencies.  For example, you can simultaneously serve one backend that
+uses legacy Tensorflow 1 and another backend that uses Tensorflow 2.
```

Comment:
Should add somewhere in here that the two environments must be of the same python and ray versions.